### ISSUE TITLE
bug: animation not like Framer motion 

### ISSUE THREAD

**Issue by reslear** [tmg0/hero-motion#100]
what can be done to improve the animation ?

play: https://stackblitz.com/~/github.com/reslear/hero-motion/tree/reslear/patch-10592
i add example: https://github.com/tmg0/hero-motion/pull/99

now:

https://github.com/tmg0/hero-motion/assets/12596485/8582cd03-b256-4570-81d2-21ca2d113d6b

framer motion:

https://github.com/tmg0/hero-motion/assets/12596485/c1eeae5b-4e75-4284-aa47-5a8d681cd65f

**Comment by tmg0** [comment]
Thank you for your using this package.

This issue caused by the style `display: grid` and `place-items: center`. `grid` layout causes the position of the box changed and leading to the error of initial position calculation.

Currently I am trying some solutions (but it’s not going well). Welcome suggestions!

For the problem in the example, in current version, you can try adding a static size wrapper around the two boxes:

```vue
<template>
  <div class="wrap" style="text-align: left">
    <button layout class="button" @click="showSecond = !showSecond">Animate</button>

    <div :style="{ width: showSecond ? '96px' : '48px', height: showSecond ? '96px' : '48px' }">
      <Hero as="div" v-if="showSecond" layout-id="rectangle" class="second-element" :style="{ width: '96px', height: '96px' }" />
      <Hero v-else as="div" layout-id="rectangle" class="element" :style="{ width: '48px', height: '48px' }" />
    </div>
  </div>
</template>
```

**Comment by tmg0** [comment]
Thank you for your raising this issue.

This issue should be fixed in `v0.2.4`

Now we used transform scale instead of width and height, and we do not need a wrapper for the `Hero` component any more~

The example can be checked in [stackblitz](https://stackblitz.com/~/github.com/tmg0/hero-motion)
